pom.xml

```diff
@@ -2183,8 +2183,8 @@
         <identity.branding.preference.management.version>1.0.1</identity.branding.preference.management.version>
 
         <!-- Identity REST API feature -->
-        <identity.api.dispatcher.version>2.0.0</identity.api.dispatcher.version>
-        <identity.user.api.version>1.2.2</identity.user.api.version>
+        <identity.api.dispatcher.version>2.0.7-SNAPSHOT</identity.api.dispatcher.version>
+        <identity.user.api.version>1.2.3-SNAPSHOT</identity.user.api.version>
```

Comment:
Version number to be confirmed after merging wso2/identity-api-user#143

Reply:
bumped